include/types.h

```diff
@@ -6,11 +6,17 @@ typedef struct match_t {
     int len;
 } match_t;
 
+typedef struct which_buttons_ignore_t {
+    unsigned char how_much_symbols;
```

Comment:
This should be "how many", but I'd actually like to just name this something like "count" or "length", that's simpler and clear enough.